lib/rules/landmark-unique-matches.js

```diff
@@ -1,8 +1,22 @@
 import { isVisibleToScreenReaders } from '../commons/dom';
+import { closest } from '../core/utils';
 import { getRole } from '../commons/aria';
 import { getAriaRolesByType } from '../commons/standards';
 import { accessibleTextVirtual } from '../commons/text';
 
+/*
+ * Since this is a best-practice rule, we are filtering elements as dictated by ARIA 1.1 Practices regardless of treatment by browser/AT combinations.
+ *
+ * Info: https://www.w3.org/TR/wai-aria-practices-1.1/#aria_landmark
+ */
+const excludedParentsForHeaderFooterLandmarks = [
+  'article',
+  'aside',
+  'main',
+  'nav',
+  'section'
+].join(',');
```

Comment:
What's this doing here?

Reply:
Seems my local `develop` was out of date, fixed in [74e1cdc](https://github.com/dequelabs/axe-core/pull/4493/commits/74e1cdc1a5a23126f7cfc3d7ef7933e0088a9968)